project/pyproject.toml.jinja

```diff
@@ -28,6 +28,7 @@ pytest-cov = "^2.11.1"
 pytest-randomly = "^3.5.0"
 pytest-sugar = "^0.9.4"
 pytest-xdist = "^2.2.0"
+# safety = "^1.10.3" # install this locally outside of the venv `pip install safety --user`
```

Comment:
What are your thoughts about leaving this inline comment so when people run into the error there aware of an easy safe solution. If its in documentation i missed then i would say we should remove it entirely but seems like something that might be helpful if someone is like why is this failing on first load if there is not a warning elsewhere when running `poetry run duty check` and it hits depends checks and notices its missing

Reply:
I think it would be better to print a warning in the `check-dependencies` duty when safety is not available.

```diff
     nofail = False
     safety = which("safety")
     if not safety:
         pipx = which("pipx")
         if pipx:
             safety = f"{pipx} run safety"
         else:
-            safety = "safety"
-            nofail = True
+            ctx.run("echo Please install safety or pipx to run this task && false", title="Checking dependencies", nofail=True)
+            return
```

Reply:
@pawamoy 
As that is your code you want me to take out the comment and you can add in your idea when doing next release?